Dockerfile

```
@@ -42,6 +42,9 @@ RUN \
     # Support old init script dir name
     ln -s /etc/osg/image-{init,config}.d
 
+COPY bin/pkg-cmp-between.sh               /usr/local/bin
+COPY bin/pkg-cmp-gt.sh                    /usr/local/bin
+COPY bin/pkg-cmp-lt.sh                    /usr/local/bin
```

```suggestion
COPY bin/*               /usr/local/bin
```

Docker knows how to use simple wildcards, which allows us to reduce the number of container image layers and thus the size of the final container